Chart: Add Getting Started topic (#7613)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chart/00 Getting Started with Chart/00 Getting Started with Chart.md

#include tutorials-intro-installationnote

> DevExtreme Chart is an interactive UI component that displays data in a chart format. The component supports multiple chart [types](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/).

Reviewer: I would shorten this introductory paragraph to a single sentence. The first sentence simply says that a "chart component displays a chart", which is already implied in the name.

```suggestion
DevExtreme Chart is an interactive UI component that visualizes data using multiple [series types](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/): line, bar, bubble, candlestick, and others.
```

> DevExtreme Chart is an interactive UI component that visualizes data using multiple [series types](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/): line, bar, bubble, candlestick, and others.

Reviewer 2: ```suggestion
The DevExtreme Chart is an interactive UI component that visualizes data using multiple [series types](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/): lines, bars, bubbles, candlesticks, and more.
```

> This tutorial explains how to add a Chart to a page and configure the component's core settings.

Reviewer 2: ```suggestion
This tutorial explains how to add a Chart to a page and configure component core settings.
```
